src/components/Search/FilterDropdowns/UserSelectPopup.tsx

```diff
@@ -77,7 +89,7 @@ function UserSelectPopup({value, closeOverlay, onChange}: UserSelectPopupProps)
                 includeCurrentUser: true,
             },
         );
-    }, [options.reports, options.personalDetails]);
+    }, [options.reports, options.personalDetails, selectedOptions]);
```

Comment:
What about these warnings, do you need the `selectedOptions` here? Can you leave a comment explaning that in the code if so please